### PR TITLE
Updated 1.8 SDK to improve NMEA elevation

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

File: recipes/qgis/recipe.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ DEPS_qgis=(zlib gdal qca libspatialite libspatialindex expat gsl postgresql libz
 # DEPS_qgis=()
 
 # url of the package
-URL_qgis=https://github.com/qgis/QGIS/archive/df28e637cdddc0c6e93355a9f98ad2d12d9e6acc.tar.gz
+URL_qgis=https://github.com/qgis/QGIS/archive/94ac9f21b8b7e36d3aa9715922ad15a7bf62fdae.tar.gz
```
Comment (m-kuhn) [inline recipes/qgis/recipe.sh:11]: Which commit is that?
I assumed we either add a patch or bump the version.

Comment (nirvn) [reply recipes/qgis/recipe.sh:11]: I did an update to QGIS 3.16.3 to grap latest fixes and optimizations, on top of which we apply the patch.

Comment (m-kuhn) [reply recipes/qgis/recipe.sh:11]: Ah cool. Can we use the branch name instrad of the sha?

Comment (nirvn) [reply recipes/qgis/recipe.sh:11]: Sounds like a good idea. TIL we can do that :)

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**